src/modules/networks/networks-store.ts

```diff
@@ -30,26 +30,24 @@ export class NetworksStore extends Store<State> {
   private networkConfigs: NetworkConfig[] = [];
   private customNetworkConfigs: NetworkConfig[] = [];
   private loaderPromises: Record<string, Promise<Networks>> = {};
-  private getEthereumChainConfigs:
-    | null
-    | (() => Promise<EthereumChainConfig[] | undefined>);
   client: Client;
   testnetMode: boolean;
+  private getChainSources: null | (() => Promise<ChainConfig | undefined>);
 
   constructor(
     state: State,
     {
-      getEthereumChainConfigs,
+      getChainSources,
       client,
       testnetMode,
     }: {
-      getEthereumChainConfigs?: NetworksStore['getEthereumChainConfigs'];
+      getChainSources?: NetworksStore['getChainSources'];
```

Comment:
I was thinking about a better name and so far have come up with `"otherNetworksData"`
So this method would accordingly be named `getOtherNetworksData`

What do you think?

Reply:
I've though more about `getSavedNetworkData`.
Cause what we get here is everything we've saved about networks configs/visits/etc